run_robot_test.sh

```diff
@@ -114,6 +114,34 @@ case "$(uname -s)" in
         ;;
 esac
 
+## if we have yq installed
+if command -v yq &> /dev/null
```

Comment:
Can you make this conditional based on a command line argument instead? This would be preferable so that a developer running the script locally does not current oc context?

Example: If I am logged into a cluster as cluster-admin but the test admin user is a completely different user with a different permissions. Running this would switch my local login and require me to reset it back after each execution

Reply:
In jenkins we are using `--kubeconfig=/home/jenkins/workspace/rhods-tier2/62/kconfig`  so the different jobs don't interfere each other's session. Could this  approach work here?

Reply:
done by creating a param `--skip-oclogin` whose default value is `true`. If set to `false`, the script will try to oc login

Reply:
@jgarciao I'll investigate more on your suggestion to see if it could work, thank you!

Reply:
hm do you mean enable the login by default? I could just change the default value in the `run_robot_test.sh` script rather than setting it in the Docker file. Would it work for you?